File: src/test/fixtures.zig
const std = @import("std");

pub const fixtures = std.ComptimeStringMap([]u8, .{
    .{ "package.json", @embedFile("./fixtures/package.json") },
    .{ "tsconfig.json", @embedFile("./fixtures/tsconfig.json") },
    .{ "simple-component.js", @embedFile("./fixtures/simple-component.js") },
    .{ "simple-component.tsx", @embedFile("./fixtures/simple-component.tsx") },
    .{ "simple-component.tsx", @embedFile("./fixtures/simple-component.tsx") },
});

File: src/runtime/test_runner/harness/fixtures.rs
bun_core::comptime_string_map! {
    pub static FIXTURES: &'static [u8] = {
        b"package.json" => include_bytes!("./fixtures/package.json"),
        b"tsconfig.json" => include_bytes!("./fixtures/tsconfig.json"),
        b"simple-component.js" => include_bytes!("./fixtures/simple-component.js"),
        b"simple-component.tsx" => include_bytes!("./fixtures/simple-component.tsx"),
    };
}
